gemini/use-cases/document-processing/doc_parsing_with_llamaindex_and_llamaparse.ipynb

```diff
@@ -0,0 +1,841 @@
+      "source": [
+        "PROJECT_ID = \"\"  # @param {type:\"string\"}\n",
+        "LOCATION = \"\"  # @param {type:\"string\"}\n",
+        "\n",
+        "import vertexai\n",
+        "\n",
+        "vertexai.init(project=PROJECT_ID, location=LOCATION)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "f818f7127a12"
+      },
+      "source": [
+        "### Setting up the Environment\n",
+        "Install dependencies"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "c76df03a88f3"
+      },
+      "outputs": [],
+      "source": [
+        "%pip install google-cloud-aiplatform \\\n",
+        "  llama-index \\\n",
+        "  langchain-community \\\n",
+        "  llama-index-embeddings-vertex \\\n",
+        "  llama-index-llms-vertex \\\n",
+        "  termcolor \\\n",
+        "  llama-index-core -q"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "44d71c554004"
+      },
+      "source": [
+        "Set up imports"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 5,
+      "metadata": {
+        "id": "fb7e65420d45"
+      },
+      "outputs": [],
+      "source": [
+        "from langchain.text_splitter import RecursiveCharacterTextSplitter\n",
+        "from llama_index.core import Settings, SimpleDirectoryReader, VectorStoreIndex\n",
+        "from llama_index.core.base.response.schema import Response\n",
+        "from llama_index.core.extractors import KeywordExtractor, QuestionsAnsweredExtractor\n",
+        "from llama_index.core.ingestion import IngestionPipeline\n",
+        "from llama_index.core.node_parser import LangchainNodeParser\n",
+        "from llama_index.embeddings.vertex import VertexTextEmbedding\n",
+        "from llama_index.llms.vertex import Vertex\n",
+        "from llama_index.vector_stores.vertexaivectorsearch import VertexAIVectorStore\n",
+        "from llama_parse import LlamaParse"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8d7a288c44bb"
+      },
+      "source": [
+        "Generate credentials"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 6,
+      "metadata": {
+        "id": "5e5cb3d58df5"
+      },
+      "outputs": [],
+      "source": [
+        "import google.auth\n",
+        "import google.auth.transport.requests\n",
+        "\n",
+        "# credentials will now have an api token\n",
+        "credentials = google.auth.default(quota_project_id=\"genai-noabe\")[0]\n",
+        "request = google.auth.transport.requests.Request()\n",
+        "credentials.refresh(request)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "0e9455150d49"
+      },
+      "outputs": [],
+      "source": [
+        "gemini_embedding_model = VertexTextEmbedding(\n",
```

Comment:
technically this isn't a Gemini Embedding model

Reply:
Updated to embedding_model.